nautobot/core/templates/inc/object_details_advanced_panel.html

```diff
@@ -57,6 +59,23 @@
             </tbody>
         </table>
     </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Data Provenance</strong>
+        </div>
+        <table class="table table-hover panel-body attr-table">
+            <tbody>
+                <tr>
+                    <td>Created By</td>
```

Comment:
Looks weird to me to have "Created" in one panel and "Created By" in a different panel. Wonder if we can/should move the "Created" and "Last Updated" entries into the "Data Provenance" panel as well - any thoughts @lampwins?